site/content/en/guides/producer/setters/_index.md

```diff
@@ -87,6 +87,51 @@ spec:
   replicas: 3 # {"$kpt-set":"replicas"}
 ```
 
+#### Targeting fields using the path
+
+To limit the fields targeted by the setter, use the `--field` flag to provide
+either the name of the field, the full path to the field, or a partial (suffix)
+path to the field. Only fields that match the path will be targeted by the setter.
+
+```sh
+# create a setter named "replicas" and but only target the field name replicas in the spec
+kpt cfg create-setter hello-world/ replicas 3 --field="spec.replicas"
+```
+
+```yaml
+# deployment-foo.yaml
+kind: Deployment
+metadata:
+  name: foo
+  annotations:
+    replicas: 3
+spec:
+  replicas: 3 # {"$kpt-set":"replicas"}
+```
+
+The path is always just the names of the properties on the path to the field,
+regardless whether the field is nested inside a sequence.
+
+```sh
+# create a setter named "mountName" and only target the name of the volume mount
+kpt cfg create-setter hello-world/ mountName nginx --field="spec.containers.volumeMounts.name"
+```
+
+```yaml
+# deployment-nginx.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+spec:
+  containers:
+  - name: nginx
```

Comment:
Interesting part is to show another element in the array here and whether its property is selected.

Reply:
If both field path and value match, then they will be selected.

Reply:
So in this example, without using the `--field` flag, several other fields would have been selected.